### ISSUE TITLE
Help text is not displayed

### ISSUE THREAD

**Issue by Jamesking56** [timothyasp/nova-color-field#14]
When adding help text to the element in Nova, the help text is not displayed inside of the admin area.

e.g.:

```php
Color::make(__('Colour'), 'colour')
    ->help(__('Help text here that should be shown.'))
    ->slider()
```

The text inside the `help` call is not shown on the forms.

**Comment by timothyasp** [comment]
Thanks for the report - i'll take a look, but happy to merge a pull that fixes this.

**Comment by timothyasp** [comment]
This has been fixed in v2.0.1
